test/semantics/omp-clause-validity01.f90

```diff
@@ -48,12 +48,19 @@
   enddo
   !$omp end parallel
 
-  !ERROR: 'LASTPRIVATE(A)' not allowed in OMP PARALLEL
-  !ERROR: 'NUM_TASKS(4)' not allowed in OMP PARALLEL
-  !ERROR: 'INBRANCH' not allowed in OMP PARALLEL
+  !ERROR: 'LASTPRIVATE' clause is not allowed on the PARALLEL directive
+  !ERROR: 'NUM_TASKS' clause is not allowed on the PARALLEL directive
+  !ERROR: 'INBRANCH' clause is not allowed on the PARALLEL directive
   !$omp parallel lastprivate(a) NUM_TASKS(4) inbranch
   do i = 1, N
      a = 3.14
   enddo
   !$omp end parallel
+
+  !ERROR: At most one 'NUM_THREADS' clause can appear on the PARALLEL directive
+  !$omp parallel num_threads(2) num_threads(4)
+  do i = 1, N
+     a = 3.14
+  enddo
+  !$omp end parallel
```

Comment:
It would be good to have a comment in the file that references the section of the standard being tested.

Reply:
okay!

Reply:
Addressed in commit [cb89744](cb89744746186bcf120d8ca62a791e2778366b9b).